src/utils/helpers.js

```diff
@@ -74,9 +74,16 @@ export const isMinimumServerVersion = (currentVersion, minMajorVersion = 0, minM
     // Dot version is equal
     return true;
 };
+
+let sentMessageToRemoveBackwardsCompatibility = false;
 export const sendMessageToRemoveBackwardsCompatibility = (currentVersion, minVersion) => {
     if (process.env.NODE_ENV !== 'production' && currentVersion > minVersion) { //eslint-disable-line no-process-env
+        if (sentMessageToRemoveBackwardsCompatibility) {
```

Comment:
Checking inside the `NODE_ENV` test so that this gets compiled out altogether in production.